eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -163,10 +162,9 @@ public final class TranspileMojo extends SafeMojo {
     @Override
     public void exec() {
         final Collection<ForeignTojo> sources = this.scopedTojos().withShaken();
-        final Optimization optimization = this.transpilation();
         final int saved = new Threaded<>(
             sources,
-            tojo -> this.transpiled(tojo, optimization)
+            tojo -> this.transpiled(tojo, this.transpilation())
```

Comment:
@ArtemGet here, instead creating just one object you will create new object on every cycle iteration. So I believe this change is unnecessary

Reply:
Aggreed, reverted changes.